project_generator/tool.py

```diff
@@ -113,13 +113,12 @@ def get_value(self, tool, key):
     def get_supported(self):
         return self.TOOLS.keys()
 
-def export(exporter, data, tool, env_settings):
+def export(exporter, project_workspace, tool, env_settings):
```

Comment:
Is it really a project workspace being passed in or is it a dict? - this is where refacoring would be nice so we only need to pass in the project and could get the tool and env_settings from its instance inside this function. We have so many nice class variables but pass them around instead of using them.